homeassistant/components/ecovacs/quality_scale.yaml

```diff
@@ -0,0 +1,95 @@
+rules:
+  # Bronze
+  config-flow: done
+  test-before-configure: done
+  unique-config-entry: done
+  config-flow-test-coverage: done
```

Comment:
I am missing a test where we try to set up the same account again

Reply:
Implemented with https://github.com/home-assistant/core/pull/133572